workflow/rules/freebayes_vc.smk

```diff
@@ -1,22 +1,95 @@
 import snakemake
 
-sample_annotation = config['sample_annotation']
+if config['doSim'] == True:
```

Comment:
put duplicated logic into functions, and do not repeat any block of codes with more than one line